<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newProfileWindow</class>
 <widget class="QMainWindow" name="newProfileWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>132</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>328</width>
    <height>132</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>328</width>
    <height>132</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ویرایش پروفایل</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/edit.png</normaloff>../icons/edit.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="mainGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>7</y>
      <width>311</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>ویرایش</string>
    </property>
    <widget class="QPushButton" name="cancelPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>86</y>
       <width>84</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>انصراف</string>
     </property>
    </widget>
    <widget class="QPushButton" name="savePushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>86</y>
       <width>84</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>ذخیره</string>
     </property>
    </widget>
    <widget class="QLabel" name="profileNameLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>نام جدید:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="profileNameLineEdit">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>21</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="dateLabel">
     <property name="geometry">
      <rect>
       <x>209</x>
       <y>51</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>تاریخ اجرا:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dateLineEdit">
     <property name="geometry">
      <rect>
       <x>122</x>
       <y>53</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string>0000/00/00</string>
     </property>
     <property name="text">
      <string>1400//</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
